src/api/app/services/staging/staging_project_creator.rb

```diff
@@ -0,0 +1,45 @@
+module Staging
+  class StagingProjectCreator
+    def initialize(staging_project_names, staging_workflow)
+      @staging_project_names = staging_project_names
+      @staging_workflow = staging_workflow
+    end
+
+    def create
```

Comment:
Is there any reason to distinguish here between create and initialize? It looks like this is basically the same, or not?